### PR TITLE
Slideshow: IE11 Issues

### PR DESCRIPTION
**Pull request by jeffersonrabb**
#### Changes proposed in this Pull Request

This PR resolves two IE11 issues:

- Block fails with Syntax Error
- Buttons (previous/next/pause/play) have no background images

The Syntax Error appears to be an issue with importing Swiper, which is resolved by explicitly specifying the file path in the import. Here are two Swiper issues which are possibly related:

https://github.com/nolimits4web/swiper/issues/2282
https://github.com/nolimits4web/Swiper/issues/2237

The button appearance issue is resolved by URL encoding the SVGs. This was done using this utility: https://yoksel.github.io/url-encoder/. 

#### Testing instructions

- Check out this branch on local Jetpack instance
- Build blocks with SDK
- Create new post in IE11 and add Slideshow block. 
- Verify the block works as expected in the editor, and that the buttons appear normal.
- Publish the post and verify the block in the view.

Note: One approach to testing in IE11 is to expose your local instance at a public URL using [Ngrok](https://ngrok.com/), then use [BrowserStack](https://www.browserstack.com/) to test in a virtual machine running IE11.

Fixes https://github.com/Automattic/wp-calypso/issues/31232

### CODE REVIEW

File: client/gutenberg/extensions/slideshow/slideshow.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 /**
  * External dependencies
  */
+import { __ } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
```
Comment (simison) [inline client/gutenberg/extensions/slideshow/slideshow.js]: Odd how lack of this didn't fatal previously. 🤔

Comment (simison) [inline client/gutenberg/extensions/slideshow/slideshow.js]: Checked that this file doesn't get imported to `view.js` 👍 

For the measure also checked that slideshow depends only on `lodash` on view side: https://github.com/Automattic/jetpack/blob/f024195cb6f208603c6da239431c5c5bc481f3a9/extensions/blocks/slideshow/slideshow.php#L26-L28

**Comment by simison** [comment]
> The Syntax Error appears to be an issue with importing Swiper, which is resolved by explicitly specifying the file path in the import. Here are two Swiper issues which are possibly related:

To elaborate little on this;

`swiper` module has these entires at `package.json`: 
```
  "jsnext:main": "dist/js/swiper.esm.bundle.js",
  "main": "dist/js/swiper.js",
  "module": "dist/js/swiper.esm.bundle.js",
```

And we'll need `swiper.js` instead of `swiper.esm.bundle.js`, which fatals in IE11:

image

with offending code around the error being `,clearTimeout:function(){}}:window;class n{constructor(e){`.

File: client/gutenberg/extensions/slideshow/slideshow.js (hunk 1 above)
Comment (simison) [reply client/gutenberg/extensions/slideshow/slideshow.js]: Oh right, we just didn't hit the code path where it is used. I missed that; apparently, I tested the other similar error and not that.

**Comment by simison** [comment]
Been reading if there's some way in Webpack to force import from `main` but didn't figure it out yet. Some reading:
- https://webpack.js.org/api/module-methods/#import-
- https://webpack.js.org/guides/code-splitting/
- https://medium.com/webpack/webpack-4-import-and-commonjs-d619d626b655

**Comment by sirreal** [comment]
> Been reading if there's some way in Webpack to force import from main but didn't figure it out yet. 

Webpack is resolving the module based on [`resolve.mainFields`](https://webpack.js.org/configuration/resolve/#resolvemainfields), which defaults as follows is browser targets:

```js
module.exports = {
  //...
  resolve: {
    mainFields: ['browser', 'module', 'main']
  }
};
```

Swiper has these in its `package.json`:

```json
{
  "main": "dist/js/swiper.js",
  "jsnext:main": "dist/js/swiper.esm.bundle.js",
  "module": "dist/js/swiper.esm.bundle.js",
}
```

`browser` is not found, `module` is, so `dist/js/swiper.esm.bundle.js` is resolved when the `swiper` module is imported.

**Comment by simison** [comment]
FYI @Automattic/team-calypso as this was an interesting example.

File: client/gutenberg/extensions/slideshow/slideshow.js (hunk 1 above)
Comment (simison) [reply client/gutenberg/extensions/slideshow/slideshow.js]: Went in here https://github.com/Automattic/wp-calypso/pull/31192

Comment (jeffersonrabb) [reply client/gutenberg/extensions/slideshow/slideshow.js]: This mistake was caused by two PRs crossing paths. The i18n import was removed in https://github.com/Automattic/wp-calypso/pull/31175 after https://github.com/Automattic/wp-calypso/pull/31192 was already open. 31175 landed first, then 31192 soon after. My mistake was not running one last test on 31192—it just looked like a clean merge. I'm also unclear why there wasn't a lint error.

Comment (jeffersonrabb) [reply client/gutenberg/extensions/slideshow/slideshow.js]: FYI discussion of the `lodash` dependency here: https://github.com/Automattic/wp-calypso/pull/30789

**Comment by jeherve** [comment]
Cherry-picked to the release branch in 45e563be625a034d05c72b91e501ac946f5c425e

**Comment by blowery** [comment]
Related to https://github.com/Automattic/wp-calypso/pull/30196.

**Pull request merged**